src/components/ProgressBarContainer.ts

```diff
@@ -140,12 +151,37 @@ export default class ProgressBarContainer extends Component<ProgressBarContainer
 
     private updateValues(mxObject?: mendix.lib.MxObject): ProgressBarContainerState {
         return {
+            displayTextAttributeValue: mxObject ? mxObject.get(this.props.displayTextAttribute) as string : "",
             maximumValue: this.getValue(this.props.maximumValueAttribute, this.defaultMaximumValue, mxObject),
             progressValue: this.getValue(this.props.progressAttribute, undefined, mxObject),
             themeStyle: this.getBarStyle (mxObject)
         };
     }
 
+    private getDisplayText = (mxObject?: mendix.lib.MxObject) => {
+        const { displayTextAttribute } = this.props;
+
+        if (mxObject) {
+            this.setState({
+                displayTextAttributeValue: mxObject.get(displayTextAttribute) as string
+            });
+        } else {
+            this.setState({
+                displayTextAttributeValue: ""
+            });
```

Comment:
useless if else statement could be done with ternary operator
`mxObject ? mxObject.get(displayTextAttribute) as string : ""`

Reply:
Done